### PR TITLE
Added infos about cli's "--target" flag regarding package bundling

### PR DESCRIPTION
**Pull request by isthatcentered**
Allows for a more concious choice of export.

I use parcel to bundle npm libraries, discovering "dependencies" are not bundled in "node" export mode was of great use :)

### CODE REVIEW

File: src/i18n/en/docs/cli.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ Available in: `serve`, `watch`, `build`
 parcel build entry.js --target node
 ```
 
+⚠️ Target `node` and `electron` will not bundle package.json's `dependencies` (but will include `devDependencies`)
```
Comment (DeMoorJasper) [inline src/i18n/en/docs/cli.md]: This can be overwritten using `--bundle-node-modules`. Please add this flag as well and mention it in this warning

Comment (isthatcentered) [reply src/i18n/en/docs/cli.md]: Sure, happy to do so. 

The last two commits address your request :)

File: src/i18n/en/docs/cli.md [hunk 2]
```diff
@@ -91,8 +91,22 @@ Available in: `serve`, `watch`, `build`
 parcel build entry.js --target node
 ```
 
+⚠️ Target `node` and `electron` will not bundle package.json's `dependencies` (but will include `devDependencies`). This behavior can be overriden using --bundle-node-modules flag (see below)
```
Comment (DeMoorJasper) [inline src/i18n/en/docs/cli.md]: Could you use a link for the reference to `#force-node-modules-bundling`

Comment (isthatcentered) [reply src/i18n/en/docs/cli.md]: Oh yeah, duh xD Done

**Pull request merged**